pom.xml

```diff
@@ -524,9 +523,25 @@ under the License.
         <scala.version>2.12.17</scala.version>
         <spark.version.short>3.4</spark.version.short>
         <parquet.version>1.13.1</parquet.version>
-        <additional.3_3.test.source>spark-3.3-plus</additional.3_3.test.source>
-        <additional.3_4.test.source>spark-3.4</additional.3_4.test.source>
-        <shims.minorVerSrc>spark-3.4</shims.minorVerSrc>
```

Comment:
These shims are removed?

Reply:
These are redundant and the same as default properties.